index.js

```diff
@@ -9,14 +9,23 @@ function search(dir) {
   var context = path.resolve(dir)
   if (!fs.existsSync(context)) return []
   var contents = fs.readdirSync(context)
-  return contents.map(function(name) {
+  var scopedRoots = []
+  var results = contents.filter(function (name) {
+    var isBin = name === ".bin"
```

Comment:
Since `.bin` is a special folder (never an npm package itself), this is filtering it from the results to be processed in the following `map` step. Not filtering it adds extra work and could lead to other issues since the map step is expecting these folders to be packages (AFAIK).